bin/alias.bat

```diff
@@ -1,27 +1,31 @@
 @echo off
 
 set ALIASES=%CMDER_ROOT%\config\aliases
+setlocal
+:: handle quotes within command definition, e.g. quoted long file names
+set _x="%*"
+set _x=%_x:"=%
 
-if ["%*"] == [""] echo Use /? for help & echo. & goto :p_show
+:: check command usage
+if ["%_x%"] == [""] echo Use /? for help & echo. & goto :p_show
```

Comment:
Probably no need to replace `%*` with `%_x%` here because if  `["%_x%"] == [""]` then `["%*"] == [""]` will be true as well.